### PR TITLE
Remove autogluon_contrib_nlp + Remove text tutorials + Revise tutorial structure

### PR DESCRIPTION
**Pull request by sxjscience**
*Issue #, if available:*

*Description of changes:*

- [x] Remove `autogluon_contrib_nlp` in the dependencies of autogluon.text. Also, mark it as deprecated and revise the document accordingly
- [x] Disable progress bar in huggingface when launching the tutorials
- [x] Reorganize the tutorials of AutoGluon Multimodal.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/text_prediction/index.rst [hunk 1]
```diff
@@ -1,34 +1,5 @@
...
-
-.. toctree::
-   :maxdepth: 1
-   :hidden:
-
-   beginner
-   customization
-   multilingual_text
+AutoGluon Text is now deprecated and merged into [AutoGluon Multimodal](https://auto.gluon.ai/stable/tutorials/multimodal/index.html).
+Check AutoGluon Multimodal for more details on how to automatically tune foundational models for text prediction problem.
```
Comment (Innixma) [inline docs/tutorials/text_prediction/index.rst:5]: Grammar: "for text prediction problems"

File: text/src/autogluon/text/text_prediction/predictor.py [hunk 2]
```diff
@@ -67,10 +67,17 @@ def __init__(
         warn_if_exist : bool, default = True
             Whether to raise warning if the specified path already exists.
         """
+        warnings.warn(
+            f"AutoGluon TextPredictor is deprecated in v0.6 and will be removed in 0.7. "
```
Comment (Innixma) [inline text/src/autogluon/text/text_prediction/predictor.py:71]: `v0.7`

File: docs/tutorials/text_prediction/index.rst (hunk 1 above)
Comment (sxjscience) [reply docs/tutorials/text_prediction/index.rst]: fixed.

File: text/src/autogluon/text/text_prediction/predictor.py (hunk 2 above)
Comment (sxjscience) [reply text/src/autogluon/text/text_prediction/predictor.py]: fixed.

**Review by zhiqiangdon — APPROVED** [review]
LGTM!

**Review by bryanyzhu — APPROVED** [review]
Thank you for the re-organization, it looks great.

**Pull request merged**